packages/fx-core/resource/package.nls.json

```diff
@@ -781,6 +781,15 @@
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
   "core.NewOfficeAddinOptionVSC.label": "Start with an Outlook add-in",
   "core.NewOfficeAddinOptionVSC.detail": "Use the Teams Toolkit to create a new Office add-in application.",
+  "core.summary.actionDescription": "Action %s%s",
+  "core.summary.lifecycleDescription": "Running lifecycle stage: %s(%s step(s) in total) The following actions will be executed %s",
```

Comment:
Q: not sure about 'Running', do we really 'run' something for each lifecycle stage? or this is to say: "Running through lifecycle stages"?

Reply:
It looks like this in ttk.
![image](https://user-images.githubusercontent.com/18023393/217470155-990048bc-2f14-40e9-b4ce-cac2a6a72ed0.png)
What should I change it to?